package.json

```diff
@@ -2,10 +2,10 @@
   "name": "@optbot/pytools",
   "version": "0.0.0",
```

Comment:
I disagree. The service doesn't exist yet beyond its initial commit, which copies template. We might as well get as much out of the numbers as we can. This is intended to be the first usable version, which in computer means starting with 0. The [trello card](https://trello.com/c/sFMJmRFl) is labeled accordingly.